README.md

```diff
@@ -74,4 +74,4 @@ Creating a new release
 6. Once your PR is merged, wait for the tests to pass on CircleCI for develop
 7. Create a "Release" on GitHub with the proper tag version and notes from the changelog.
     - The tag should be identical to the version in `package.json`
-8. Run `yarn run publish:local`
\ No newline at end of file
+8. Run `yarn run publish:local`
```

Comment:
I went through the Github.com flow; guessing this is an automatic endline change artifact here.